src/lib/DpApi.js

```diff
@@ -4,18 +4,21 @@ import { v4 as uuid } from 'uuid'
 
 let currentProcedureId = null
 let jwtToken = null
+let csrfToken = null
 
 if (typeof dplan !== 'undefined') {
   if (hasOwnProp(dplan, 'procedureId')) {
     currentProcedureId = dplan.procedureId
   }
   if (hasOwnProp(dplan, 'jwtToken')) {
     jwtToken = dplan.jwtToken
+    csrfToken = dplan.csrfToken
   }
 }
 
 const apiDefaultHeaders = {
-  'X-JWT-Authorization': 'Bearer ' + jwtToken
+  'X-JWT-Authorization': 'Bearer ' + jwtToken,
+  'x-csrf-token': csrfToken
```

Comment:
why is the new header only added to apiDefaultHeaders, not api2defaultHeaders (which is used on all 2.0 routes)?